### PR TITLE
Add digest article

### PR DESCRIPTION
**Pull request by advaypal**

### CODE REVIEW

File: _posts/2016-10-04-digest-text-generation-with-markov-models.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: "Digest: Text Generation with Markov Models"
+date: 2016-10-04 21:41
+author: Anonymous
```
Comment (burnflare) [inline _posts/2016-10-04-digest-text-generation-with-markov-models.md]: Nameless? :)

**Comment by burnflare** [comment]
Hmm why don't the images load? https://github.com/nushackers/nushackers-site/pull/27/files?short_path=976e48d#diff-976e48d6aa52216dc69ef16f41d3968b

**Comment by advaypal** [comment]
The images aren't loading because I used the site.url variable to refer to them. They're loading fine on the blog.

**Comment by advaypal** [comment]
The original links have code snippets/link to code

**Pull request merged**